### PR TITLE
docs(issue labeling): explain impact label

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Add new section for the `impact:*` labels

## Context

We need to explain what the `impact:*` labels are used for, and what the goal is for having them. Follow up after adding impact labels to our repository.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/development/issue-labeling.md [hunk 1]
```diff
@@ -88,6 +88,24 @@ At a high level: critical = needs immediate fix, important = to be prioritized a
...
+    impact:large
+
+</details>
+
+Use these to assign a impact level to an issue.
+Do we mean?
+
+- Impact on usability for end users
+- Impact on maintainers, like "amount of hassle/work this takes to complete"
+- Impact as risk of ...
```
Comment (HonkingGoose) [inline docs/development/issue-labeling.md:107]: Need help from the maintainers to explain what the labels are for.

The label descriptions on GitHub are:

- `impact:small` Change has a small impact
- `impact:medium` Change has a moderate impact on users/use cases
- `impact:large` Change has potentially large impact on users/use cases

Comment (rarkins) [reply docs/development/issue-labeling.md:107]: At the moment impact == risk.

It's not meant to mean "amount of work"

Comment (HonkingGoose) [reply docs/development/issue-labeling.md:107]:
```suggestion
Impact means risk to the end users or their use cases.
It does _not_ mean "amount of work for the maintainers".
```

Is this correct? Feel free to edit the suggestion so it matches what you mean. 😉

File: docs/development/issue-labeling.md [hunk 2]
```diff
@@ -88,6 +88,21 @@ At a high level: critical = needs immediate fix, important = to be prioritized a
...
+    <summary>Impact</summary>
+
+    impact:small
+    impact:medium
+    impact:large
+
+</details>
+
+Use these to assign a impact level to an issue.
+Impact means risk to the end users or their use cases.
```
Comment (rarkins) [inline docs/development/issue-labeling.md:103]:
```suggestion
Impact means risk to the end users or their use cases.
It's used to identify which changes can be made relatively quickly versus those which require great care before merging, due to their chance of negatively impacting a wide number of users if there's a bug.
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**